jung-algorithms/src/test/java/edu/uci/ics/jung/layout/spatial/FRLayoutsTimingTest.java

```diff
@@ -59,6 +59,10 @@ public void testFRBH() {
     // using the same random seed each time for repeatable results from each test.
     layoutAlgorithmTwo.setRandomSeed(0);
     doTest(layoutAlgorithmTwo);
+    try {
+      Thread.sleep(1000);
+    } catch (InterruptedException ex) {
+    }
   }
```

Comment:
sometimes the iterator version is so slow that the test finishes before it reports the time.

Reply:
This hack needs to go. Stay tuned....